public-images/public-images-reference/artifacts.rst

```diff
@@ -0,0 +1,1495 @@
+
+Virtual Hard Disk (VHD)
+~~~~~~~~~~~~~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``.vhd.tar.gz``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64-azure.vhd.tar.gz``
+   * - Format description
+     - A VHD is a file format used by virtualisation software to store virtual hard disk images.
+   * - Use cases
+     - VHD files are used in cloud environments for storing VM disk images. They allow easy deployment and scaling of virtual machines. You can use a ``.vhd`` file to create an instance based on a pre-configured image, ensuring consistency across multiple deployments.
+
+Other files
+-----------
+This section includes information on checksums, GPG signatures, changelogs and manifest files. These files help verify the integrity and authenticity of the images, provide details on changes between versions and list all included packages.
+
+Changelogs
+~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``.image_changelog.json``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64.daily.20240612.20240622.image_changelog.json``
+   * - Format description
+     - JSON-formatted changelogs detailing the changes in the respective image builds.
+   * - Data format
+     -
+      | - **summary:**
+      |   - **snap:**
+      |     - **added:** Lists newly added snap packages.
+      |     - **removed:** Lists removed snap packages.
+      |     - **diff:** Lists snap packages that have changed.
+      |   - **deb:**
+      |     - **added:** Lists newly added deb packages.
+      |     - **removed:** Lists removed deb packages.
+      |     - **diff:** Lists deb packages that have changed.
+      | - **diff:**
+      |   - **deb:** Detailed information on changed deb packages:
+      |     - **name:** The name of the package.
+      |     - **from_version:** Previous version details, including the source package name and version.
+      |     - **to_version:** New version details, including the source package name and version.
+      |     - **cves:** Common Vulnerabilities and Exposures fixed.
+      |     - **launchpad_bugs_fixed:** IDs of fixed Launchpad bugs.
+      |     - **changes:** A list of changes with details like CVEs, change logs, package name, version,
+      |       urgency, distributions, author and date.
+      |     - **notes:** Additional notes, if any.
+      |   - **snap:** Detailed information on changed snap packages (same structure as deb).
+      | - **added:** Lists newly added deb and snap packages.
+      | - **removed:** Lists removed deb and snap packages.
+      | - **notes:** General notes regarding the changelog.
+      | - **from_series:** The starting series name.
+      | - **to_series:** The target series name.
+      | - **from_serial:** The starting serial number.
+      | - **to_serial:** The target serial number.
```

Comment:
These four - from_series, to_series, from_serial and to_serial are not very clear. Could you explain it a bit more or maybe add an example for them?

Reply:
I'll make it more explicit and add an example. The `from` are the series/serial from the previous image, the `to` are the current image.